public/src/js/models/config/front.js

```diff
@@ -221,9 +222,44 @@ export default class ConfigFront extends BaseClass {
             alert('You must choose a group');
             return;
         }
+        if (this.hasDuplicatePlatformSpecificCollection()) {
+            alert('Cannot share platform-specific collections with other fronts');
+            return;
+        }
         return persistence.front.update(this);
     }
 
+    hasDuplicatePlatformSpecificCollection() {
+        const dups = this.collections.items().filter(c => isPlatformSpecificCollection(c.meta.platform()) && c.parents().length > 1);
```

Comment:
Would using find here instead of filter better describe what is happening?

Reply:
good point

Reply:
This is weird, `find` works when I run the app, but when I run the tests it's `undefined`...which suggests it's not a real array!? I'll leave it as `filter` as that works

Reply:
ahh that makes sense, will do